packages/manager/modules/pci/src/projects/project/private-networks/vrack/add/index.js

```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-ovh-uirouter-layout';
+import '@ovh-ux/ng-ovh-swimming-poll';
+
+import component from './add.component';
+import routing from './add.routing';
+import service from './add.service';
+
+const moduleName = 'ovhManagerPciPrivateNetworksVrackAdd';
+
+angular
+  .module(moduleName, [
+    'ngTranslateAsyncLoader',
```

Comment:
Missing import statement for both `ngTranslateAsyncLoader` and `pascalprecht.translate`.